cmd/mt-store-cat/main.go

```diff
@@ -0,0 +1,340 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"math"
+	"os"
+	"runtime"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+
+	"github.com/raintank/dur"
+	"github.com/raintank/metrictank/api"
+	"github.com/raintank/metrictank/mdata"
+	"github.com/raintank/metrictank/mdata/chunk"
+	"github.com/rakyll/globalconf"
+	"gopkg.in/raintank/schema.v1"
+)
+
+const tsFormat = "2006-01-02 15:04:05"
+
+var (
+	GitHash = "(none)"
+
+	// flags from metrictank.go, globals
+	showVersion = flag.Bool("version", false, "print version string")
+	confFile    = flag.String("config", "/etc/raintank/metrictank.ini", "configuration file path")
+
+	// flags from metrictank.go, Cassandra
+	cassandraAddrs               = flag.String("cassandra-addrs", "localhost", "cassandra host (may be given multiple times as comma-separated list)")
+	cassandraKeyspace            = flag.String("cassandra-keyspace", "raintank", "cassandra keyspace to use for storing the metric data table")
+	cassandraConsistency         = flag.String("cassandra-consistency", "one", "write consistency (any|one|two|three|quorum|all|local_quorum|each_quorum|local_one")
+	cassandraHostSelectionPolicy = flag.String("cassandra-host-selection-policy", "roundrobin", "")
+	cassandraTimeout             = flag.Int("cassandra-timeout", 1000, "cassandra timeout in milliseconds")
+	cassandraReadConcurrency     = flag.Int("cassandra-read-concurrency", 20, "max number of concurrent reads to cassandra.")
+	//cassandraWriteConcurrency    = flag.Int("cassandra-write-concurrency", 10, "max number of concurrent writes to cassandra.")
+	cassandraReadQueueSize = flag.Int("cassandra-read-queue-size", 100, "max number of outstanding reads before blocking. value doesn't matter much")
+	//cassandraWriteQueueSize      = flag.Int("cassandra-write-queue-size", 100000, "write queue size per cassandra worker. should be large engough to hold all at least the total number of series expected, divided by how many workers you have")
```

Comment:
i was wondering the same thing. this was my way of signifying that all options are the standard cassandra store options, except the commented out ones aren't relevant here.